hack/Dockerfile.golang

```diff
@@ -27,16 +27,16 @@ FROM base-${TARGETARCH}
 ARG TARGETARCH
 ARG GO_VERSION
 RUN tar -C /usr/local -xzf go${GO_VERSION}.linux-${TARGETARCH}.tar.gz && \
-    rm go${GO_VERSION}.linux-${TARGETARCH}.tar.gz
+	rm go${GO_VERSION}.linux-${TARGETARCH}.tar.gz
 
 # install cgo-related dependencies
 RUN set -eux; \
 	dnf install -y \
-		g++ \
-		gcc \
-		glibc-devel \
-		make \
-		pkg-config \
+	g++ \
+	gcc \
+	glibc-devel \
+	make \
+	pkg-config \
 	; \
```

Comment:
intentional formatting changes ?

Reply:
Yep intentional, it is put in a separate commit :-)

Reply:
What formatter are you using for this? I prefer the previous indention, but it is not super important to me, and I know it is pain to fight against automatic formatters. :wink:

Reply:
haha, it is the default one that comes with the Dockerfile plugin for vscode.